modin/data_management/functions/binary_function.py

```diff
@@ -11,16 +11,68 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""Module houses Binary functions builder class."""
+
 import numpy as np
 import pandas
 
 from .function import Function
 
 
 class BinaryFunction(Function):
+    """Builder class for Binary functions."""
+
     @classmethod
-    def call(cls, func, *call_args, **call_kwds):
+    # FIXME: spread `*call_args` and `**call_kwds` into an actual function arguments.
+    def call(cls, func, *call_args, **call_kwds):  # noqa: PR02
```

Comment:
`PR02: unknown parameter 'preserve_labels'`, here we're documenting parameters which are hidden in `**kwargs`

Reply:
In this case, let's add it to the params if it is usually expected and documented. It's ok to do here IMO.

Reply:
got rid of all of the `PR02 noqa`s at the last commit